src/setupTests.ts

```diff
@@ -5,7 +5,7 @@ SPDX-License-Identifier: BSD-3-Clause
 // Temporary hack to suppress error
 // https://github.com/facebookincubator/create-react-app/issues/3199
 import * as Enzyme from 'enzyme';
-import Adapter from 'enzyme-adapter-react-16';
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```

Comment:
Can you put a note in a comment above this import explaining why we're using a 3rd-party adapter (maybe linking to the Enzyme deprecation) so we know to keep an eye on its continued development?

Reply:
Yes. Good call.

Reply:
Fixed. Thanks.